<launch>
  <arg name="camera" default="naoqi_driver/camera" />

  <node pkg="extrinsic_calibration" type="extrinsic_calibration" name="extrinsic_calibration_node" output="screen" cwd="node">
    <param name="depth_img_topic" value="/$(arg camera)/ir/image_raw" />
    <param name="rgb_img_topic" value="/$(arg camera)/front/image_raw" />

    <param name="camera_info_depth_topic" value="/$(arg camera)/depth/camera_info" />
    <param name="camera_info_rgb_topic" value="/$(arg camera)/front/camera_info" />

    <param name="images_folder" value="/home/crossing-pepper-1/Documents/calibration_images" />

    <param name="pattern_cols" value="7" />
    <param name="pattern_rows" value="5" />
    <param name="pattern_size" value="0.03" />
  </node>

</launch>
